src/flexbox.rs

```diff
@@ -106,17 +106,17 @@ struct AlgoConstants {
     inner_container_size: Size<f32>,
 }
 
-impl Taffy {
+impl<T> Taffy<T> {
```

Comment:
I don't like the way that this design ends up polluting the whole tree with generics. That said, a default type argument would help a lot.